bundle/bin/rke2-uninstall.ps1

```diff
@@ -139,6 +122,11 @@ Get-Process -ErrorAction Ignore -Name "containerd*" | ForEach-Object {
     $_ | Stop-Process -ErrorAction Ignore -Force
 }
 
+Get-Process -ErrorAction Ignore -Name "wins*" | ForEach-Object {
+    Write-LogInfo "Stopping process $($_.Name) ..."
+    $_ | Stop-Process -ErrorAction Ignore -Force
```

Comment:
You would consider a function like this. https://github.com/phillipsj/wins/blob/feature/rancher-install-script/rancher-installer.ps1#L474-L490